main.py

```diff
@@ -63,23 +75,48 @@ def send_email(subject, body):
 
 @app.route('/')
 def index():
-    return render_template("base.html")
+    assignments = get_weekly_assignments()
+    return render_template("base.html", assignments=assignments)
 
 @app.route('/notify', methods=['POST'])
 def notify():
     assignments = get_weekly_assignments()
     if assignments:
         email_subject = "Upcoming Assignments Due This Week"
-        email_body = "<h1>Upcoming Assignments Due This Week:</h1>"
+        email_body = """
```

Comment:
@MCdev92 Can you move email_body content to a separate file (example email_content.py) that way we don't clutter our main.py file.

Reply:
update, email body moved to separate file: 
<img width="1402" alt="image" src="https://github.com/MCdev92/CanvasNotification_Webapp/assets/110692284/5eefc8dc-2191-471f-9a38-c107f7309808">